Dockerfile

```diff
@@ -6,7 +6,11 @@ LABEL MAINTAINER="BBT Software AG <devadmin@bbtsoftware.ch>"
 ENV DEBIAN_FRONTEND=noninteractive
 RUN echo "APT::Get::Assume-Yes \"true\";" > /etc/apt/apt.conf.d/90assumeyes
 
-RUN apt-get update && \
+RUN mkdir /azp && \
+    groupadd -r azure --gid=999 && \
```

Comment:
IMHO `azure` is a confusing name (beside that Azure DevOps is too), why not calling user and group `azp`?